### PR TITLE
(fix) Adding crop size validation.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: server/apps/archive/archive_crop.py [hunk 1]
```diff
@@ -88,6 +88,11 @@ def _validate_aspect_ratio(self, crop, doc):
         """
         width = doc['CropRight'] - doc['CropLeft']
         height = doc['CropBottom'] - doc['CropTop']
+
+        if width < crop['width'] or width < crop['height']:
```
Comment (amagdas) [inline server/apps/archive/archive_crop.py]: `height < crop['height']` ?

Comment (sivakuna-aap) [inline server/apps/archive/archive_crop.py]: Is the second comparison `width < crop['height']` supposed to be against height or width?

**Comment by mdhaman** [comment]
yes. me bad....

File: server/apps/archive/archive_crop.py (hunk 1 above)
Comment (mdhaman) [inline server/apps/archive/archive_crop.py]: yes. me bad

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

File: server/apps/archive/archive_crop.py [hunk 2]
```diff
@@ -88,6 +88,11 @@ def _validate_aspect_ratio(self, crop, doc):
         """
         width = doc['CropRight'] - doc['CropLeft']
         height = doc['CropBottom'] - doc['CropTop']
+
+        if width < crop['width'] or height < crop['height']:
+            raise SuperdeskApiError.badRequestError(
+                message='Wrong crop size. Minimum crop size is {}x{}.'.format(crop['width'], crop['height']))
```
Comment (plamut) [inline server/apps/archive/archive_crop.py]: If you prefer, you can shorten the error message string with the following (the line appears a bit lengthy):
```python
'Wrong crop size. Minimum crop size is {width}x{height}.'.format(**crop)
```

**Comment by plamut** [comment]
:+1:

File: server/apps/archive/archive_crop.py (hunk 2 above)
Comment (mdhaman) [inline server/apps/archive/archive_crop.py]: yes agree. will remember that.

**Pull request merged**